cmd/operator/main.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/GoogleCloudPlatform/prometheus-engine/pkg/operator"
 )
 
+const (
+	defaultTLSDir = "/etc/tls/private"
```

Comment:
question: how did you come up with this directory? Is there a standard TLS directory?

Reply:
It's not exactly standard, but this is what prometheus-operator uses.
https://github.com/prometheus-operator/prometheus-operator/blob/aedb6cacda6ffed64b43893b1f4fc12b4677db8a/pkg/server/server.go#L38